pkg/daemon/update.go

```diff
@@ -718,6 +718,12 @@ func (dn *Daemon) reboot(rationale string) error {
 	}
 	dn.logSystem("machine-config-daemon initiating reboot: %s", rationale)
 
+	// Now that everything is done, avoid delaying shutdown.
+	if dn.installedSigterm {
+		signal.Reset(syscall.SIGTERM)
+		dn.installedSigterm = false
```

Comment:
this isn't really needed right? we get here sequentially through `handleNodeUpdated` or the very first sync so we don't care about this variable anymore just before rebooting

Reply:
Correct, but I'm trying to keep in mind future support for live updates (where we'd install/deinstall the term handler without rebooting).

Reply:
ack